src/components/ui/toaster.tsx

```diff
@@ -1,35 +1,45 @@
-"use client"
+"use client";
 
-import { useToast } from "@/hooks/use-toast"
+import React, { useMemo } from "react";
+import { useToast } from "@/hooks/use-toast";
+import { useRouter } from "next/navigation";
 import {
   Toast,
   ToastClose,
   ToastDescription,
   ToastProvider,
   ToastTitle,
   ToastViewport,
-} from "@/components/ui/toast"
+} from "@/components/ui/toast";
 
 export function Toaster() {
-  const { toasts } = useToast()
+  const { toasts } = useToast();
+  const router = useRouter();
+  const toast = toasts[0];
+
+  useMemo(() => {
+    if (toast?.path && toast?.isSuccess) {
```

Comment:
I am not sure what's going on here. Why do we need this logic in toast component?

Reply:
Sir @Monfernape  In this context, we’re utilizing the shadcn Toast. When the toast is shown and its success status is true, it will update the path as needed; note that the path is optional.